src/tests/multihost/ad/test_ad_misc.py

```diff
@@ -191,15 +191,12 @@ def test_0002_improved_use_negative_sid_for_sid_lookup(
           1. domain log file should show that the SID of non-existing does
              not exist in negative cache
         """
-        client = sssdTools(multihost.client[0])
+        adjoin(membersw='adcli')
+        client = sssdTools(multihost.client[0], multihost.ad[0])
         domain_name = client.get_domain_section_name()
-        multihost.client[0].service_sssd('stop')
+        client.backup_sssd_conf()
         domain_section = f'domain/{domain_name}'
```

Comment:
now unused variable.

Reply:
i was trying different approaches here.  unused var is removed.